inc/cleantalk-public-validate.php

```diff
@@ -421,3 +431,24 @@ function ct_contact_form_validate_postdata()
 
     return null;
 }
+
+add_filter('apbct__filter_post', 'apbct__filter_form_data', 10);
```

Comment:
It would be great to use this one https://www.php.net/manual/ru/function.array-filter.php instead of apply_filter.
It gives us cross-platform usage.